app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -71,28 +70,61 @@ def usps_proofer
 
       def create_usps_enrollment(enrollment)
         pii = user_session[:idv][:pii]
+        address = pii.address1
+        address += " #{pii.address2}" unless pii.address2.blank?
```

Comment:
how about
```suggestion
        address = [pii.address1, pii.address2].select(&:present?).join(' ')
```

Reply:
Added in [7240116](https://github.com/18F/identity-idp/pull/6629/commits/72401167466c1c1ef890f85c7e6675f4be8987b5)